legend-engine-language-pure-modelManager-sdlc/src/main/java/org/finos/legend/engine/language/pure/modelManager/sdlc/SDLCLoader.java

```diff
@@ -180,7 +180,9 @@ else if (context.sdlcInfo instanceof AlloySDLC)
                     }
                     else
                     {
-                        throw new RuntimeException("The following PackageableElementPointers:" + missingPaths.toString() + " do not exist in the project data loaded from the metadata server");
+                        throw new RuntimeException("The following entities:" + missingPaths + " do not exist in the project data loaded from the metadata server. " +
```

Comment:
Please throw an EngineException instead of a RuntimeException. Also, add a category to the exception.

Right now we have code all over the place that throws RuntimeException. This makes it hard to analyse the exceptions as all of them are RuntimeException.

Reply:
Changed to EngineException!  ;)